service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -1,24 +1,92 @@
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class CaseManagementOrderIssuedEventHandler {
-    private final CaseManagementOrderCaseLinkNotificationHandler caseLinkNotificationHandler;
-    private final CaseManagementOrderDocumentLinkNotificationHandler documentLinkNotificationHandler;
+    private final ObjectMapper mapper;
+    private final InboxLookupService inboxLookupService;
+    private final NotificationService notificationService;
+    private final RepresentativeService representativeService;
+    private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+    private final CafcassLookupConfiguration cafcassLookupConfiguration;
+    private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
 
     @EventListener
     public void sendEmailsForIssuedCaseManagementOrder(final CaseManagementOrderIssuedEvent event) {
         EventData eventData = new EventData(event);
 
-        caseLinkNotificationHandler.sendNotifications(eventData);
-        documentLinkNotificationHandler.sendNotifications(eventData,
-            event.getDocumentContents());
+        sendToLocalAuthority(eventData);
+        sendToCafcass(eventData);
+        sendToRepresentatives(eventData, DIGITAL_SERVICE);
+        sendToRepresentatives(eventData, EMAIL);
+
+        // TODO
+        // Refactor issuedOrderAdminNotificationHandler to use Java object for template so we can avoid passing byte
+        // to method
+        issuedOrderAdminNotificationHandler.sendToAdmin(eventData, event.getDocumentContents(), CMO);
+    }
+
+    private void sendToLocalAuthority(final EventData eventData) {
+        final IssuedCMOTemplate localAuthorityNotificationParameters = caseManagementOrderEmailContentProvider
+            .buildCMOIssuedNotificationParameters(eventData.getCaseDetails(), DIGITAL_SERVICE);
+
+        final String email = inboxLookupService.getNotificationRecipientEmail(eventData.getCaseDetails(),
+            eventData.getLocalAuthorityCode());
+
+        notificationService.sendEmail(CMO_ORDER_ISSUED_NOTIFICATION_TEMPLATE, email,
+            localAuthorityNotificationParameters, eventData.getReference());
+    }
+
+    private void sendToCafcass(final EventData eventData) {
+        final IssuedCMOTemplate cafcassParameters =
+            caseManagementOrderEmailContentProvider.buildCMOIssuedNotificationParameters(
+                eventData.getCaseDetails(), EMAIL);
+
+        final String cafcassEmail = cafcassLookupConfiguration.getCafcass(eventData.getLocalAuthorityCode()).getEmail();
+
+        notificationService.sendEmail(CMO_ORDER_ISSUED_NOTIFICATION_TEMPLATE, cafcassEmail, cafcassParameters,
+            eventData.getReference());
+    }
+
+    private void sendToRepresentatives(final EventData eventData, RepresentativeServingPreferences servingPreference) {
+        CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
```

Comment:
I think we should call sendToRepresentativesByServedPreference from RepresentativeNotificationService in here instead as it is doing the same thing; you need to pass in parameters but the parameters don't change depending on the representative so you can generate them and then just pass them in

Reply:
Agree with this change but `sendToRepresentativesByServedPreference` takes `final Map<String, Object> templateParameters` which when using java object for templates would require me to call `toMap`on my template object before the `NotificationService` tries to call toMap again. 

It would require a refactor of `RepresentativeNotificationService.sendToRepresentativesByServedPreference` to accept an overload to handle generic `notifyData`. Out of scope for this ticket but happy to make a tech deb and clean this and others up.